lib/src/core/injection_container.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:get_it/get_it.dart';
+import 'package:zone_app/src/features/devices/data/local/bluetooth_device_local_source.dart';
+import 'package:zone_app/src/features/devices/data/repository/device_repository_impl.dart';
+import 'package:zone_app/src/features/devices/domain/repository/device_repository.dart';
+import 'package:zone_app/src/features/devices/domain/usecase/connect_device_usecase.dart';
+import 'package:zone_app/src/features/devices/domain/usecase/disconnect_device_usecase.dart';
+import 'package:zone_app/src/features/devices/domain/usecase/read_values_usecase.dart';
+import 'package:zone_app/src/features/devices/domain/usecase/search_devices_usecase.dart';
+import 'package:zone_app/src/features/devices/domain/usecase/write_values_usecase.dart';
+import 'package:zone_app/src/features/devices/presentation/bloc/device_list_cubit.dart';
+import 'package:zone_app/src/features/devices/presentation/bloc/device_selected_cubit.dart';
+import 'package:zone_app/src/features/devices/presentation/bloc/read_values_cubit.dart';
+import 'package:zone_app/src/features/devices/presentation/bloc/write_value_cubit.dart';
+
+final sl = GetIt.instance;
```

Comment:
Change variable name to injector or serviceLocator
```suggestion
final injector = GetIt.instance;
```